### PR TITLE
Modify rules S6320;S6416;S6555;S6707;S6976 Update examples

### PR DESCRIPTION
**Pull request by sallaigy**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6555/java/rule.adoc [hunk 1]
```diff
@@ -8,14 +8,11 @@ A reference to ``++null++`` should never be dereferenced/accessed. Doing so will
...
-  return condition ? "name" : null;
-}
-
-public boolean isNameEmpty() {
-  return getName().length() == 0; // Noncompliant; the result of getName() could be null, but isn't null-checked
+public boolean isNameEmpty(String givenName, String familyName) {
+  if (givenName != null && familyName != null) {
+    return givenName.isEmpty() && familyName.isEmpty();
+  }
+  return familyName.isEmpty(); // Noncompliant; familyName may be null
```
Comment (chrislain-razafimahefa-sonarsource) [inline rules/S6555/java/rule.adoc:15]: There is no 'compliant' section for this rule?

**Review by chrislain-razafimahefa-sonarsource — APPROVED** [review]
LGTM! S6555 does not have a compliant section!?

File: rules/S6555/java/rule.adoc (hunk 1 above)
Comment (sallaigy) [reply rules/S6555/java/rule.adoc:15]: The original rule example didn't have one either, so I did not include one here either. Would you prefer that we add one?

Comment (chrislain-razafimahefa-sonarsource) [reply rules/S6555/java/rule.adoc:15]: True, I also saw that the original one did not have one. It seems weird not to have one when all the other rules do, so I think we should, if you have no objections.

**Review by chrislain-razafimahefa-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**